arrows/tests/test_video_input.h

```diff
@@ -170,10 +170,19 @@ void test_seek_frame( kwiver::vital::algo::video_input& vi )
   // Test valid seeks after invalid seeks
   valid_seeks = {11, 32, 21, 43};
 
-  for (auto requested_frame : in_valid_seeks)
+  for (auto requested_frame : valid_seeks)
   {
-    EXPECT_FALSE( vi.seek_frame( ts, requested_frame) );
-    EXPECT_NE( requested_frame, ts.get_frame() );
+    EXPECT_TRUE( vi.seek_frame( ts, requested_frame ) )
+      << "seek_frame should return true for requested_frame " << requested_frame;
+
+    auto img = vi.frame_image();
+
+    ASSERT_TRUE( !! img );
```

Comment:
Nitpicking, but should there be a space after !!  ? And maybe add a comment about what this is checking since the other tests with EXPECT_TRUE have outputs that are descriptive

Reply:
This is just copy-pasted from lines 146-159, so I decided not to fix the style. There are lots of issues with this file that need addressing, but I might leave that for another PR. I only included this change because it was clearly functionally wrong and very simple to fix.